### PR TITLE
add Tilt support

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ smi-adapter-istio-5ffcm8fqm               1/1       Running     0          20s
 
 ## Contributing
 
-Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on contributing to the specification.
+Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on contributing to the specification and setting up your development environment.
```
Comment (lachie83) [inline README.md]:
```suggestion
Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on contributing to this repository and setting up your development environment.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,3 +67,10 @@ eval $(minikube docker-env)
 operator-sdk build devimage
 kubectl -n istio-system delete pod -l 'name=smi-adapter-istio'
 ```
+
+## Developing Using Tilt
+- Install [Tilt](https://docs.tilt.dev/install.html)
+- Replace `deislabs/smi-adapter-istio` in [manifest](deploy/kubernetes-manifests.yaml) and [Tiltfile](Tiltfile) with your own image name i.e. `<dockeruser>/smi-adapter-istio`
```
Comment (lachie83) [inline CONTRIBUTING.md]: Can you point the contributor at the steps to build and publish their own container before this step?

File: Tiltfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Deploy: tell Tilt what YAML to deploy
+k8s_yaml('deploy/kubernetes-manifests.yaml')
+
+custom_build(
+  'deislabs/smi-adapter-istio',
```
Comment (lachie83) [inline Tiltfile]: Does this need to change to the user defined container?

File: Tiltfile [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  'operator-sdk build $EXPECTED_REF',
+  ['./pkg', './cmd'],
+  tag='latest',
+  live_update=[
+    sync('pkg', '/go/src/github.com/deislabs/smi-adapter-istio/pkg'),
```
Comment (lachie83) [inline Tiltfile]: Does this path get autodetected in the case /go/src is NOT your $GOPATH can you make it use the value of $GOPATH instead?

File: CONTRIBUTING.md (hunk 2 above)
Comment (michelleN) [reply CONTRIBUTING.md]: done

File: Tiltfile (hunk 4 above)
Comment (michelleN) [reply Tiltfile]: This line says copy the `pkg` directory relative to the Tilefile (in your local dev environment) to the path `/go/src/github.com/deislabs/smi-adapter-istio/pkg` inside of your container filesystem. That path doesn't need to change bc it is fixed.

File: Tiltfile (hunk 3 above)
Comment (michelleN) [reply Tiltfile]: Yes, there are instructions in the CONTRIBUTING.md that address that.

**Comment by michelleN** [comment]
@lachie83 I believe I address your concerns. Please take another look and let me know if there is anything else. Thank you.

**Pull request merged**